### DISCUSSION TITLE
See full response body in timeline view

### DISCUSSION THREAD

**Discussion by fractalic** [Kong/insomnia#3696] [Ideas]
I'd love to be able to see complete request and response objects in the timeline view. I often find myself copying the timeline record out into another program for archiving, and it slows things down when I have to switch back to the Preview tab to see the response body.

This is very similar to these past issues, which added the ability to show Request chunks in the timeline view:
* https://github.com/Kong/insomnia/issues/174
* https://github.com/Kong/insomnia/issues/1262

This is an example of the response I see in the timeline. It'd be great if there was an option to show the response itself close to where the `Received chunk` message is.
```
< HTTP/1.1 200 OK
< Date: Tue, 22 Jun 2021 17:37:30 GMT
< Content-Type: application/json; charset=utf-8
< Content-Length: 114


* Received 114 B chunk
* Connection #1 to host
```

**Comment by chamby** [comment]
This would be useful for me as well.

```
< HTTP/2 404 
< server: nginx/1.17.3
< content-type: application/json
< vary: Accept-Encoding
< cache-control: no-cache, private
< date: Mon, 26 Jul 2021 17:40:04 GMT
< x-ratelimit-limit: 60
< x-ratelimit-remaining: 58


* Received 92 B chunk
* Connection #0 to host stg.redacted.com left intact
```

However, the body has some useful information in it that isn't part of the timeline.

```
{
  "error": {
    "message": "The authenticated user has no products set up yet."
  },
  "status_code": 404
}
```

**Comment by dimitropoulos** [comment]
Overall this is a great discussion so far (thanks @fractalic and @chamby for the high-quality info!).

As a developer, the first thing I'd do is write a test (or, I guess, manually conduct a smoke test, heh..) where I have a request with a body and a diff for a timeline showing what is now and what the desired state should be.  That's that best way to keep the ball rolling on this.
